cfy_manager/components/prometheus/config/node_exporter.service

```
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus exporter for hardware and OS metrics
+Documentation=https://github.com/prometheus/node_exporter
+Wants=network-online.target
+After=network-online.target[Service]
```

```suggestion
After=network-online.target
```
i assume thats a typo